src/static/js/ace2_inner.js

```diff
@@ -2930,36 +2920,22 @@ function Ace2Inner(){
     //console.log("%o %o %s", rep.selStart, rep.selEnd, rep.selFocusAtStart);
   }
 
-  function ifAttribIsFormattingUpdateStyleButtonState(attribName, hasStyleOnRepSelection) {
-    if (attributeIsFormattingStyle(attribName)) {
-      var $formattingButton = parent.parent.$('[data-key="' + attribName + '"]').find('a');
-      $formattingButton.toggleClass(SELECT_BUTTON_CLASS, hasStyleOnRepSelection);
-    }
+  function updateStyleButtonStateWhenAttribIsFormatting(attribName, hasStyleOnRepSelection) {
```

Comment:
this function needs to be renamed too, now you don't check if attrib is a formatting one :)